### PR TITLE
Add new sensor type DPT 9.009 "air_flow"

### PR DESCRIPTION
**Pull request by rtreichl**
## Description

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] The documentation has been adjusted accordingly
- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] The changes are documented in the changelog (docs/changelog.md)

### CODE REVIEW

File: xknx/dpt/dpt_2byte_float.py [hunk 1]
```diff
@@ -173,6 +173,15 @@ class DPTPartsPerMillion(DPT2ByteFloat):
     unit = "ppm"
 
 
+class DPTAirFlow(DPT2ByteFloat):
+    """DPT 9.009 DPT_Value_AirFlow."""
+
+    dpt_main_number = 9
+    dpt_sub_number = 9
+    value_type = "m³/h"
```
Comment (farmio) [inline xknx/dpt/dpt_2byte_float.py:181]: This is used for lookup eg. in HA sensor configuration (`type: ...`). Having special characters (or whitespace) in there seems a bit awkward. 

I'd call it `air_flow` or `airflow` - or whatever the Knx specs say. It also has to be unique.

**Review by farmio — CHANGES_REQUESTED** [review]

**Comment by farmio** [comment]
To satisfy Codecov consider doing a rebase from current `main`.

File: xknx/dpt/dpt_2byte_float.py (hunk 1 above)
Comment (rtreichl) [reply xknx/dpt/dpt_2byte_float.py:181]: you are right... 
i'll change it to "air_flow"

**Review by farmio — APPROVED** [review]
Thank you very much 👍

**Pull request merged**